com.unity.multiplayer.mlapi/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -335,93 +341,49 @@ private static void OnSceneUnloadServer(Guid switchSceneGuid)
+                    var ownerClientId = reader.ReadUInt64Packed();
+                    var hasParent = reader.ReadBool();
+                    ulong? parentNetworkId = null;
 
-                    for (int i = 0; i < newObjectsCount; i++)
+                    if (hasParent)
                     {
-                        bool isPlayerObject = reader.ReadBool();
-                        ulong networkId = reader.ReadUInt64Packed();
-                        ulong ownerClientId = reader.ReadUInt64Packed();
-                        bool hasParent = reader.ReadBool();
-                        ulong? parentNetworkId = null;
-
-                        if (hasParent)
-                        {
-                            parentNetworkId = reader.ReadUInt64Packed();
-                        }
-
-                        ulong prefabHash = reader.ReadUInt64Packed();
-
-                        Vector3? position = null;
-                        Quaternion? rotation = null;
-                        if (reader.ReadBool())
-                        {
-                            position = new Vector3(reader.ReadSinglePacked(), reader.ReadSinglePacked(), reader.ReadSinglePacked());
-                            rotation = Quaternion.Euler(reader.ReadSinglePacked(), reader.ReadSinglePacked(), reader.ReadSinglePacked());
-                        }
-
-                        var networkObject = NetworkManager.Singleton.SpawnManager.CreateLocalNetworkObject(false, 0, prefabHash, ownerClientId, parentNetworkId, position, rotation);
-                        NetworkManager.Singleton.SpawnManager.SpawnNetworkObjectLocally(networkObject, networkId, true, isPlayerObject, ownerClientId, objectStream, false, 0, true, false);
-
-                        var bufferQueue = NetworkManager.Singleton.BufferManager.ConsumeBuffersForNetworkId(networkId);
-
-                        // Apply buffered messages
-                        if (bufferQueue != null)
-                        {
-                            while (bufferQueue.Count > 0)
-                            {
-                                BufferManager.BufferedMessage message = bufferQueue.Dequeue();
-                                NetworkManager.Singleton.HandleIncomingData(message.SenderClientId, message.NetworkChannel, new ArraySegment<byte>(message.NetworkBuffer.GetBuffer(), (int)message.NetworkBuffer.Position, (int)message.NetworkBuffer.Length), message.ReceiveTime, false);
-                                BufferManager.RecycleConsumedBufferedMessage(message);
-                            }
-                        }
+                        parentNetworkId = reader.ReadUInt64Packed();
                     }
-                }
-            }
-            else
-            {
-                var networkObjects = UnityEngine.Object.FindObjectsOfType<NetworkObject>();
-                NetworkManager.Singleton.SpawnManager.ClientCollectSoftSyncSceneObjectSweep(networkObjects);
 
```

Comment:
This is the primary single path to loading a scene with EnableSceneManagement enabled.